library/ord-tree.lisp

```diff
@@ -390,7 +400,9 @@ The result tree may be in an intermediate state with a double-black node."
       ;; nodes with two subtrees
       ((Branch clr left _ right)
        (let (Tuple new-left new-pivot) = (remove-max left))
-       (bubble-double-black clr new-left new-pivot right))))
+       (bubble-double-black clr new-left new-pivot right))
+      (_
```

Comment:
Same as previous; `DoubleBlackEmpty` is not yet possible because we haven't inserted a double-black node yet.

Reply:
It looks like this one is also missing `Empty`

Reply:
Ah, yes, that's unreachable because `remove-leaving-double-black` is called by `remove-without-as-black` only in the case of a non-empty tree. In that case, this should probably be a catch-all pattern like
```
(_ (error "Attempt to `remove-leaving-double-black` on an empty tree"))
```